napari/_qt/widgets/qt_keyboard_settings.py

```diff
@@ -117,7 +117,7 @@ def __init__(
         layout.addWidget(
             QLabel(
                 trans._(
-                    "To clean shortcut use Backspace or Delete. To set one of this as shortcut, first clean previous shortcut."
+                    "To edit, double-click the keybinding. To unbind a key, use Backspace or Delete. To set Backspace or Delete, first unbind."
```

Comment:
```suggestion
                    "To edit, double-click the keybinding. To unbind a shorcut, use Backspace or Delete. To set Backspace or Delete, first unbind."
```

maybe this will be better?

Reply:
Does one unbind a shortcut or a key? I'm honestly not sure. I think a key is bound to a shortcut?

Reply:
is breaking this into 2 lines a good idea? is this what is forcing the window to be so wide?
<img width="1169" alt="image" src="https://user-images.githubusercontent.com/76622105/189123935-79061aa8-8774-4ae6-8784-cfb1ac0a4190.png">

Reply:
> Does one unbind a shortcut or a key?

`Ctrl+Y` is a shortcut but not a key. `V` is a shortcut and a key. 

> is breaking this into two lines a good idea
May be. Long text may prevent resizing the settings window.

Reply:
Breaking into 3 lines doesn't change the width. :shrug:

Reply:
Can you fix your suggest: "shorcut" to "shortcut" ?

Reply:
done